cocotb/config.py

```diff
@@ -97,6 +99,50 @@ def help_vars_text():
     return helpmsg
 
 
+def lib_name(interface, simulator):
+
+    interface_name = interface.lower()
+    supported_interfaces = ["vpi", "vhpi", "fli"]
+    if interface_name not in supported_interfaces:
+        raise ValueError("Wrong interface used. Supported: " + ", ".join(supported_interfaces))
+
+    simulator_name = simulator.lower()
+    supported_sims = ["icarus", "questa", "modelsim", "ius", "xcelium", "vcs", "ghdl", "riviera", "activehdl", "cvc"]
+    if simulator not in supported_sim:
+        raise ValueError("Wrong simulator name. Supported: " + ", ".join(supported_sim))
+
+    if simulator_name == "questa":
+        simulator_name = "modelsim"
+
+    if simulator_name == "xcelium":
+        simulator_name = "ius"
+
+    if simulator_name == "cvc":
+        simulator_name = "modelsim"
+
+    if simulator_name in ["riviera", "activehdl"]:
+        simulator_name = "aldec"
```

Comment:
This doesn't allow `aldec` in the list of simulator names. Is this deliberate?

Reply:
We have this:
https://github.com/cocotb/cocotb/blob/29845d6a0e7300f03a9fdc0d562d4551fbb86003/cocotb/share/makefiles/Makefile.deprecations#L9-L12

Reply:
Perhaps we should have separate `simulator_name` and `library_name` variables to make that distinction clearer.